LICENSES_DEPENDENCIES

```diff
@@ -1,129 +1,127 @@
-png                           | 0.12.0   | Apache-2.0, MIT
-proc-macro2                   | 0.4.9    | Apache-2.0, MIT
-quote                         | 0.6.3    | Apache-2.0, MIT
-rand                          | 0.3.18   | Apache-2.0, MIT
-rayon                         | 0.8.2    | Apache-2.0, MIT
-rayon-core                    | 1.3.0    | Apache-2.0, MIT
-redox_syscall                 | 0.1.31   | MIT
+num-traits                    | 0.2.8    | Apache-2.0, MIT
+num_cpus                      | 1.10.0   | Apache-2.0, MIT
+numtoa                        | 0.1.0    | MIT OR Apache-2.0
+opaque-debug                  | 0.2.2    | MIT OR Apache-2.0
+pest                          | 2.1.1    | Apache-2.0, MIT
+pest_derive                   | 2.1.0    | Apache-2.0, MIT
+pest_generator                | 2.1.0    | Apache-2.0, MIT
+pest_meta                     | 2.1.1    | Apache-2.0, MIT
+png                           | 0.14.1   | Apache-2.0, MIT
+proc-macro2                   | 0.4.30   | Apache-2.0, MIT
+quote                         | 0.6.12   | Apache-2.0, MIT
+rayon                         | 1.0.3    | Apache-2.0, MIT
+rayon-core                    | 1.4.1    | Apache-2.0, MIT
+redox_syscall                 | 0.1.54   | MIT
 redox_termios                 | 0.1.1    | MIT
-scoped_threadpool             | 0.1.8    | MIT
+scoped_threadpool             | 0.1.9    | MIT
 scopeguard                    | 0.3.3    | Apache-2.0, MIT
-sha-1                         | 0.7.0    | Apache-2.0, MIT
-strsim                        | 0.7.0    | MIT
-syn                           | 0.14.4   | Apache-2.0, MIT
-termion                       | 1.5.1    | MIT
-textwrap                      | 0.10.0   | MIT
-tiff                          | 0.2.1    | MIT
+sha-1                         | 0.8.1    | MIT OR Apache-2.0
+strsim                        | 0.8.0    | MIT
+syn                           | 0.15.34  | Apache-2.0, MIT
+termion                       | 1.5.2    | MIT
+textwrap                      | 0.11.0   | MIT
+tiff                          | 0.2.2    | MIT
 typenum                       | 1.10.0   | Apache-2.0, MIT
 ucd-trie                      | 0.1.1    | Apache-2.0, MIT
-unicode-width                 | 0.1.4    | Apache-2.0, MIT
+unicode-width                 | 0.1.5    | Apache-2.0, MIT
 unicode-xid                   | 0.1.0    | Apache-2.0, MIT
-vec_map                       | 0.8.0    | Apache-2.0, MIT
-winapi                        | 0.2.8    | MIT
-winapi                        | 0.3.5    | Apache-2.0, MIT
-winapi-build                  | 0.1.1    | MIT
+vec_map                       | 0.8.1    | Apache-2.0, MIT
+winapi                        | 0.3.7    | Apache-2.0, MIT
 winapi-i686-pc-windows-gnu    | 0.4.0    | Apache-2.0, MIT
 winapi-x86_64-pc-windows-gnu  | 0.4.0    | Apache-2.0, MIT
 
------BEGIN adler32 1.0.2 LICENSES-----
-The MIT License (MIT)
-
-Copyright (c) 2015 Remi Rampin
+-----BEGIN adler32 1.0.3 LICENSES-----
```

Comment:
adler I

Reply:
issue in #111 
resolved by https://github.com/foresterre/combostew/pull/20, #114